### DISCUSSION TITLE
Homatics Beta Testing Group

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#514] [Announcements]
![image](https://github.com/user-attachments/assets/9c8745e9-7e7c-4b03-8ed7-62928b2c78fc)

It's time to test Homatics devices, and I'd love to have YOU as a beta tester! Follow this thread and you'll be notified right away when new versions are ready, and you'll be first in line to share your input before launch!

I have already documented many of the ADB and remote commands which can be found [here](https://github.com/PRProd/HA-Firemote/wiki/Homatics-Box-R-4K-Plus)

*Tagging @Pro100x3mal as a known interested party*
<br>

## To Do List
 - [x] Release [first beta version](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0b1) for testing
 - [x] Receive and evaluate community feedback
 - [x] Complete adding Homatics commands in the existing apps list
 - [x] Publish [release candidate](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0b3)
 - [x] Update Documentation
 - [x] Release

## Known Limitations / Issues
 - ~~Visible Device Name feature is not yet ready~~
 - ~~The actual color of the LED activity light is a mystery to me - What do you have?~~
 - Not all of the buttons on my personal physical remote seem to have a usable function, so I'm unsure if I've mapped everything correctly.  Please let me know if a button doesn't do what it should

**Comment by Pro100x3mal** [comment]
Hi! Great job! Just tested it on my Homatics Box R 4k Plus. I have a remote without number buttons. Works great except for a few things:

1. The Device Name is not positioned correctly on the remote control image itself.
2. The color of the LED indicator on my remote is green if the control is via bluetooth, and red if the control is via IR. In your integration, this is not the case at the moment
3. If the tv-box is configured to control the volume of the TV via IR, then in your integration this does not work - when you press the volume control button, the volume is adjusted on the tv-box itself, but not on the TV

**Reply by PRProd** [reply]
Thanks for testing @Pro100x3mal!
> The Device Name is not positioned correctly on the remote control image itself.

This is known.  I haven't completed the work for that yet, but I did acknowledge it was missing under the **Known Limitations / Issues** section above.  This will be corrected (and crossed out above) before the official launch.
<br>

> The color of the LED indicator on my remote is green if the control is via bluetooth, and red if the control is via IR. In your integration, this is not the case at the moment

Thank you for explaining this to me!!  I have seen images online with green and with red, so I wasn't sure if there was a reason for one color over another color.

Since Firemote is neither IR or Bluetooth, and since Home Assistant has no way of knowing how your physical remote is communicating with your Homatics device, I will not be able to use the correct color (red or green) based on those rules.  Do you have a preference?  I think that red looks good on the white remotes, and green looks good on the black remotes, but I could be wrong.
<br>

> If the tv-box is configured to control the volume of the TV via IR, then in your integration this does not work - when you press the volume control button, the volume is adjusted on the tv-box itself, but not on the TV

Correct.  Since Home Assistant (and therefore Firemote) has no native way if issuing IR commands, then a decision has to be made by you, the user.  You can choose to use [CEC](https://github.com/PRProd/HA-Firemote/wiki/Using-CEC-Controls) if your TV/Audio Receiver device supports it, or you can use a [button override](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#button-overrides) to program your Firemote button function the way you expect.  Either of those options will prevent the volume on the Homatics box itself from adjusting.
<br>

❤️ THANK YOU AGAIN for testing!  Please feel free to continue testing, and let me know if you find anything else.

-Doug

**Comment by PRProd** [comment]
## Beta 2 is now available
Firemote Beta version [v4.1.0b2](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0b2) is now available to download and test.

This update includes 100s of app launchers to choose from.  The [supported apps chart](https://github.com/PRProd/HA-Firemote/wiki/Firemote-App-Launchers) has been updated to reflect these changes.

This release DOES NOT yet address the device name display issue.

**Comment by PRProd** [comment]
## Release Candidate is here
Firemote beta version [v4.1.0b3](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0b3) is now available to download and test as the release candidate.  If no problems are found / reported, then this will be pushed into the production version in a few days.

Please test, and let me know!

-Doug

**Comment by PRProd** [comment]
# HOMATICS IS HERE 🎉 

Firemote version [v4.1.0](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0) has just been officially released, along with a fancy new YouTube video!  Check it out!

https://www.youtube.com/watch?v=AggYjQ4vI-U

Thank you to all that tested, and for all that shared their results.
